src/mngt/ocf_mngt_cache.c

```diff
@@ -172,7 +172,7 @@ static void __init_partitions(ocf_cache_t cache)
 
 	/* Init default Partition */
 	ENV_BUG_ON(ocf_mngt_add_partition_to_cache(cache, PARTITION_DEFAULT,
-			"unclassified", 0, PARTITION_SIZE_MAX,
+			"unclassified", 0, cache->conf_meta->cachelines,
```

Comment:
This line is part of cache start, performed before cache attach. You cannot depend on conf_meta->cachelines being correctly set here. That's the reason why PARTITION_SIZE_MAX was used. You would probably need to iterate over partitions and trim max size to conf_meta-cachelines during cache attach

Alternatively give up setting partition max size to cache size and allow partition max to be larger than cache size - see my next comment